modules/dragster/src/main/ts/ephox/dragster/detect/NoDrag.ts

```diff
@@ -1,8 +1,15 @@
 import { Fun } from '@ephox/katamari';
+import { Event, Events } from '@ephox/porkbun';
+import { DragEvents, DragState } from './DragTypes';
+
+export const NoDrag = (): DragState => {
+  const events: DragEvents = Events.create({
```

Comment:
There was a bug here, as this was `undefined` previously which in rare cases would have caused an exception in `movement` when it tried to call `dragState.events.move`.